### PR TITLE
Handling events

### PR DESCRIPTION
**Pull request by NicolasAndrehh**
#Milestone - Handling Events
This pull request includes the following changes:
- Add calculator.js file
- Add operate.js file
- Add and edit functions in order to handle events on each button
- Improve User Interface

### CODE REVIEW

File: src/components/calculator.jsx [hunk 1]
```diff
@@ -1,26 +1,53 @@
...
+        key={value}
+        value={value}
+        setResult={changeResult}
+        calculate={calculate}
+      />,
+    );
   });
 
   return buttons;
 };
```
Comment (danmainah) [inline src/components/calculator.jsx]: - [x] @NicolasAndrehh great work on your components however it's not the best practice to have two components in the same file, kindly create a file for the createButtons component and then import it into the calculator component.

File: src/App.scss [hunk 2]
```diff
@@ -1 +1,4 @@
 // App styles
+.hide {
+  display: none;
+}
```
Comment (danmainah) [inline src/App.scss:4]: - [x] @NicolasAndrehh great work on your styling however I find your App.css and App.scss files have similar content, kindly remove one file to make your code neat and apply the best practice of avoiding repetitive code in the codebase, same case for the index.css and index.scss and app.css and app.scss,

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @NicolasAndrehh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️

- [x] @NicolasAndrehh great work on your readme, however, the live demo link is broken kindly fix it, 
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by NicolasAndrehh** [comment]
Thank you for the feedback @danmainah! 😄

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @NicolasAndrehh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**